kevm-pyk/src/kontrol/__main__.py

```diff
@@ -499,9 +513,15 @@ def parse(s: str) -> list[T]:
     foundry_prove_args.add_argument(
         '--use-booster',
         dest='use_booster',
-        default=False,
+        default=True,
         action='store_true',
-        help="Use the booster RPC server instead of kore-rpc. Requires calling foundry-kompile with the '--with-llvm-library' flag",
+        help='Use the booster RPC server instead of kore-rpc.',
+    )
+    foundry_prove_args.add_argument(
+        '--no-booster',
+        dest='use_booster',
+        action='store_false',
+        help='Do not use the booster RPC server instead of kore-rpc.',
     )
```

Comment:
We should name this option `--no-use-booster`, not `--no-booster`. The convention is to prefix the exact same argument name with `no-`.

Reply:
Renamed to `--no-use-booster` in https://github.com/runtimeverification/evm-semantics/pull/2044/commits/18f4e7722736dfa9f112aceb21b2d26c6e2adb19.